src/operator/audit.ts

```diff
@@ -86,6 +86,9 @@ class AuditSubscriber<T, R> extends OuterSubscriber<T, R> {
         this.destination.error(errorObject.e);
       } else {
         this.add(this.throttled = subscribeToResult(this, duration));
+        if (this.throttled.closed) {
```

Comment:
I'm curious about this change. Wouldn't this make it so a completed `Subject` would cause this behavior? That seems incorrect.

Is this really a subscription ordering thing? What do you think @trxcllnt?